src/byte_mapping.rs

```diff
@@ -1,5 +1,30 @@
 const NIL: char = '';
 
+/// The mapping for [ASCII](https://en.wikipedia.org/wiki/ASCII)
+///
+/// This mapping uses the standard ascii character set which is composed of a
+/// 7-bit code (or 128 characters). The first 32 characters and the last one
+/// are non-printable control characters.
+pub const CODEPAGE_ASCII: &'static [char]  = &[
+//   0     1     2     3     4     5     6     7     8     9     A     B     C     D     E     F
+    NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL, // 0
```

Comment:
Sure, I could change that.. But the current NIL character shouldn't be whitespace.. This is what I have:
![image](https://cloud.githubusercontent.com/assets/455127/25691353/3fb6306c-309a-11e7-9621-152cbf1ce065.png)

So this is not what you see?